product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/03_vnet_vnet.mdx

```diff
@@ -50,7 +50,7 @@ To walk through an example in your own environment, you need:
  ![](../images/image8.png)
 
 !!! Note
-    The VPN gateway automatically creates a dedicated subnet to accommodate its gateway VMs. Ensure that your cluster's virtual network address space has sufficient IP range for the subnet to prevent errors in the virtual network. For more information, see [Plan virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-vnet-plan-design-arm#subnets).
+    The VPN gateway automatically creates a dedicated subnet to accommodate its gateway VMs. Ensure that your cluster's virtual network address space has sufficient IP range for the subnet to prevent errors in the virtual network. For more information, see [Add a subnet](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-subnet#add-a-subnet).
```

Comment:
@moiznalwalla , why is this change in here? I thought you were just replacing screenshots?

Reply:
Never mind. I see your comment in the description.